todolist_app/src/App.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { ToastContainer,toast } from "react-toastify";
+import 'react-toastify/dist/ReactToastify.css'
+
+import Todos from "./components/Todos";
+import AddTodo from './components/addTodo';
+
+import './App.css';
+
+
+class App extends React.Component {
+  state= {
+    todos:[]
+
+  };
+
+  handleDel =(title)=>{
+    toast("deleting "+title);
+
+    let deleting =setInterval(()=>{
```

Comment:
I think you want to use `setTimeout` instead of interval, interval repeats itself